modules/gapi/test/infer/gapi_infer_onnx_test.cpp

```diff
@@ -277,14 +293,19 @@ class ONNXtest : public ::testing::Test {
                                                                 in_node_dims.size()));
         }
         // Outputs Run params
-        for(size_t i = 0; i < num_out; ++i) {
-            char* out_node_name_p = session.GetOutputName(i, allocator);
-            out_node_names.push_back(std::string(out_node_name_p));
-            allocator.Free(out_node_name_p);
+        if (custom_out_names.empty()) {
+            for(size_t i = 0; i < num_out; ++i) {
+                char* out_node_name_p = session.GetOutputName(i, allocator);
+                out_node_names.push_back(std::string(out_node_name_p));
+                allocator.Free(out_node_name_p);
+            }
+        } else {
+            out_node_names.assign(custom_out_names.begin(), custom_out_names.end());
```

Comment:
`out_node_names = custom_out_names`
It would be perfect to avoid copy here, but I'm not sure that it's possible in current implementation

Reply:
Is it critical? This fixture is Frankenstein, that support all ORT session.Run functional.

Reply:
Added std::move for `custom_out_names`. `custom_out_names` - rvalue parameter.